perses/app/setup_relative_calculation.py

```diff
@@ -538,10 +534,6 @@ def run_setup(setup_options):
             _logger.info(f'\tRunning {phase} phase...')
             hss_run = hss[phase]
 
-            _logger.info(f"\t\tminimizing...\n\n")
-            hss_run.minimize()
```

Comment:
Do we really want to eliminate minimization? I think we just want to switch minimizers.

Reply:
We are still minimizing, it's just done as part of the setup. We were minimizing twice previously